docs/README.md

```diff
@@ -4,9 +4,16 @@
+| Where&nbsp;do&nbsp;I&nbsp;get&nbsp;the&nbsp;app?         | This repository is a work in progress and has not yet been medically reviewed. Until the app is ready, the general public should consult the [WHO's COVID-19 website](https://www.who.int/emergencies/diseases/novel-coronavirus-2019).                                                                                                                                                                                                                                                                                                                                                                                |
+| How can I help out?                                      | We are humbled at the outpouring of support we have received from all over the world and are eager to have you contribute and apologize that we are still in the early process of figuring out how to manage all the contributions. Please bear with us while we get an initial version out the door and split out into smaller teams working in parallel, when we can then on-board more volunteers. Until then, please keep filing issues, and fill out the [volunteer form here](https://forms.gle/FUugWvUVvMcV3dLJA) and we’ll get back in touch. You can then read the [onboarding documentation](ONBOARDING.md). |
+| There's a problem with the app - help!                   | File a report to help us improve. Choose "Bug Report" [here](https://github.com/WorldHealthOrganization/app/issues/new/choose).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| I have an idea for the app!                              | Choose "Feature Request" [here](https://github.com/WorldHealthOrganization/app/issues/new/choose).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```

Comment:
Link directly to Feature Request template?

```suggestion
| I have an idea for the app!                              | Request a feature [here](https://github.com/WorldHealthOrganization/app/issues/new?template=feature-request.md).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```

Reply:
We can't do that.  The link doesn't just have the bug report template it, it includes the labels and other content which change over time.  No idea why GitHub does it that way :(